src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/SlimefunItem.java

```diff
@@ -429,6 +429,7 @@ private void loadItemHandlers() {
             if (!handler.isPrivate()) {
                 Set<ItemHandler> handlerset = getPublicItemHandlers(handler.getIdentifier());
                 handlerset.add(handler);
+                SlimefunPlugin.getRegistry().getPublicItemHandlers().replace(handler.getIdentifier(), handlerset);
```

Comment:
Why even? This method does literally nothing.

Reply:
It does. Without this the handlerset inside the publicItemHandlers doesn't change. I know this because I tested it.